src/support/small_set.h

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright 2021 WebAssembly Community Group participants
```

Comment:
Not easily. Those headers include a bunch of other LLVM things, which may require linking changes (since atm all that LLVM code is just used for DWARF, which is optional).